### ISSUE TITLE
[Feature] Add a platform permission with a read permission of all application

### ISSUE THREAD

**Issue by haima95** [kubevela/velaux#791]
**Is your feature request related to a problem? Please describe.**

I need a platform permission with a read permission of all application. current  if I  want to create an accout which only have the read permission of all application, I have to do the operation about add the account to the application member in every application, It's too tedious and unuseful If I have 100 application

Thank you very much

**Describe the solution you'd like**
shoud have a platform permission with a read permission of all application, if the account has the permission can read all application

**Comment by barnettZQG** [comment]
@haima95 There is a question we to need discuss. Do we need to support the admin users to have permission to manage all applications?

**Comment by haima95** [comment]
> @haima95 There is a question we to need discuss. Do we need to support the admin users to have permission to manage all applications?

Yes, some small companies have deployed 10-20 projects on k8s, and everyone has the permission to view them. This is very helpful for them to understand the company's situation, especially for newly hired employees. As for whether they have permission to modify, it can be added individually to each project.
